src/security-council-mgmt/SecurityCouncilManager.sol

```diff
@@ -146,9 +148,11 @@ contract SecurityCouncilManager is Initializable, ArbitrumTimelock, ISecurityCou
     {
```

Comment:
should this function (or `removeMember() external`) check to make sure we have >= 9 total members after removal?